web/src/message_list_tooltips.js

```diff
@@ -224,6 +225,22 @@ export function initialize() {
         },
     });
 
+    message_list_tooltip(".drafts-list .recipient_row_date", {
```

Comment:
I think we use `message_list_tooltip` plugin to handle rerenders on the message lists to avoid any ghost tooltips (whose reference message element is absent). I don't see it's use in the drafts menu where we don't use the rerendering logic. So I would suggest moving this logic over to `tippyjs.ts` instead.